src/SDKs/GuestConfiguration/Management.GuestConfiguration/Microsoft.Azure.Management.GuestConfiguration.nuget.csproj

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Why is this different from `Microsoft.Azure.Management.GuestConfiguration.csproj`?